exporters/logging/build.gradle

```diff
@@ -0,0 +1,10 @@
+description = 'OpenCensus Logging Exporter'
```

Comment:
Is this library specific to tracing, or do you think we should also provide logging for stats?